common-lib/src/cmr/common/api/errors.clj

```diff
@@ -27,8 +27,8 @@
    :body {:errors ["An Internal Error has occurred."]}})
 
 (defn- keyword-path->string-path
-  "Converts a set of keyword field paths into the string equivalent field paths to return to the
-  user."
+  "Converts a set of keyword field paths into the string equivalent field paths
```

Comment:
Not a problem, you can leave these changes and continue to use 80. Is there a way we could customize Bikeshed to complain at 100 instead since that's what all the code was using up to this point? I also like that bikeshed really lived up to its name - you get the most comments for line length. :)